Backend/WebApi.Weather/HttpController.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace WebApi.Weather;
+
+internal class HttpController
+{
+    public HttpClient Client { get { return _httpClient; } }
+    private readonly HttpClient _httpClient;
+
+    public HttpController()
+    {
+        _httpClient = new HttpClient();
```

Comment:
To bardzo zły pomysł, zobacz ten film: https://www.youtube.com/watch?v=Z6Y2adsMnAA